src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -88,6 +93,7 @@ protected static void printDecoratedExceptionMessage(final PrintStream ps, final
     protected List<String> getPackageList() {
```

Comment:
Maybe add to the documentation the packages that are included at the moment, for downstream projects being aware that Picard is also there...